src/Mail/MailService.php

```diff
@@ -42,7 +44,10 @@ public function __construct(MailerInterface $mailer, EntityManagerInterface $em,
         $this->params = $params;
     }
 
-    public function message($to, string $title, string $body, array $attachments = [])
+    /**
+     * @param array<Attachment> $attachments
+     */
+    public function message(?ContactInterface $to, string $title, string $body, array $attachments = []): void
```

Comment:
The type hint for `$to` is currently incorrect, as it could be an iterable as well. Please update it.